package.json

```diff
@@ -52,14 +52,22 @@
     "commands": [
       {
         "command": "editor.showQueryPreview",
-        "title": "Show query preview"
+        "title": "Show query preview",
+        "icon": "$(book)"
       },
       {
         "command": "dbt.compile",
         "title": "dbt compile"
       }
     ],
     "menus": {
+      "editor/title": [
+        {
+          "command": "editor.showQueryPreview",
+          "group": "navigation",
+          "when": "editorLangId =~ /^sql$|^jinja-sql$|^sql-bigquery$/ && !editorReadonly"
```

Comment:
May be we should rely on `file extension` rather that `editorLangId` ? Currently we support any adapter but some files could be marked as `postgres-sql` and so on.